pep/app/pods/components/document/text/component.ts

```diff
@@ -0,0 +1,67 @@
+import { action } from '@ember/object';
+import { inject as service } from '@ember/service';
+import Component from '@glimmer/component';
+
+import ModalService from '@gavant/ember-modals/services/modal';
+import { DS } from 'ember-data';
+
+import GlossaryTerm from 'pep/pods/glossary-term/model';
+import LoadingBarService from 'pep/services/loading-bar';
+
+interface DocumentTextArgs {
+    text: XMLDocument;
+    onGlossaryItemClick: (term: string, termResults: GlossaryTerm[]) => void;
+}
+
+export default class DocumentText extends Component<DocumentTextArgs> {
+    @service store!: DS.Store;
+    @service loadingBar!: LoadingBarService;
+    @service modal!: ModalService;
+
+    get text() {
+        // const xmlimages = this.args.text.getElementsByTagName('image');
+        return this.args.text;
+    }
+
+    /**
+     * Setup listeners on document text to handle misc. actions
+     *
+     * @memberof DocumentText
+     */
+    @action
+    setupListeners() {
+        $('.document-text').on('click', (event: JQuery.TriggeredEvent) => {
```

Comment:
since you are only doing a simple click listener directly on main element here, you don't need this, and in fact you don't even need to setup manual DOM listeners at all, you can do it right in the template/ember:

```hbs
<div class="document-text" {{on "click" this.onClick}}>{{html-safe @text}}</div>
```

```ts
onClick(event: Event) {
    const attributes = event.target.attributes as NamedNodeMap;
    //...
}
```

I'm pretty opposed to bringing in all of jQuery just for a few simple things like this, especially considering how meaty it is (~90KB), and that this is a mobile-focused app (with some of our desktop-only/admin-y type apps, i'd probably not care as much).

Even if you needed to setup some more complex listeners later (i.e. "delegation" style where you listen for events on dynamically changing content), its still only a few lines of basic DOM code, that we could easily wrap up into a util function to maintain dev ergonomics and clarity of app logic just as well as if we used jquery.